app/client/components/accountoverview/accountOverviewCard.tsx

```diff
@@ -97,6 +100,8 @@ export const AccountOverviewCard = (props: AccountOverviewCardProps) => {
 		display: inline-block;
 		vertical-align: top;
 		width: calc(100% - 15ch);
+		overflow: hidden;
```

Comment:
Subscription ids are much longer for Stripe accounts than they are for Zuora accounts, this hides and ellipsises it (see screenshot)